llvm/test/CodeGen/AMDGPU/llc-pipeline.ll

```diff
@@ -1164,6 +1170,8 @@
 ; GCN-O3-NEXT:      Lazy Block Frequency Analysis
 ; GCN-O3-NEXT:      Optimization Remark Emitter
 ; GCN-O3-NEXT:      Global Value Numbering
+; GCN-O3-NEXT:    AMDGPU Preload Kernel Arguments
+; GCN-O3-NEXT:    FunctionPass Manager
```

Comment:
Can this be moved up with the other module passes, or is there some benefit to running the other function passes first